Cargo.toml

```diff
@@ -4,5 +4,7 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+git2 = "0.20"
 regex = "1"
 serde_json = "1"
+tempfile = "3.18.0"
```

Comment:
Is there a specific reason for using "3.18..0" instead of "3"?  If not, I think "3" would be better.